index.js

```diff
@@ -4,13 +4,24 @@ require("dotenv").config();
 const express = require("express");
 const path = require("path");
 const { initDatabase } = require("./lib/database");
-const { getPaste, setPaste } = require("./lib/pastes");
+const { getPaste, setPaste, getRandomId } = require("./lib/pastes");
 
 const app = express();
 
 // Parse application/json for all request
 app.use(express.json());
 
+app.get("/random", async (request, response) => {
+  try {
+    const randomId = await getRandomId();
+    const paste = await getPaste(randomId);
```

Comment:
You already have access to the paste in the previous function. `getPaste` is not necessary